### DISCUSSION TITLE
Error: Failure while executing;

### DISCUSSION THREAD

**Discussion by vascoferraz** [ohmyzsh/ohmyzsh#10464] [Help] [answered]
Hey guys,

When I execute this command: **brew style --fix** I get this error message:

**Error: Failure while executing; `patch --get=0 --force --directory=/ --strip=0` exited with 2. Here's the output:**

![screenshot](https://user-images.githubusercontent.com/10391988/144622489-83e0dfe8-aa59-41cb-9708-c17a84c8ec33.png)

Any ideas on how to solve this?

**Comment by mcornella** [comment, marked as answer]
This doesn't look like an Oh My Zsh issue, try looking in the Homebrew repository's issues.

BTW you can add screenshots directly to a GitHub issue, it's better than hosting it somewhere else.
